var/spack/repos/builtin/packages/sim-model/package.py

```diff
@@ -67,7 +67,7 @@ def _build_mods(self, mods_location, link_flag='', include_flag='', corenrn_mods
             include_flag += " -I " + str(self.spec[dep].prefix.include)
 
         include_flag += ' -DENABLE_TAU_PROFILER' if '+profile' in self.spec else ''
-        output_dir = os.path.basename(self.neuron_archdir)
+        output_dir = os.path.basename(spec.neuron_archdir)
```

Comment:
```suggestion
        output_dir = os.path.basename(self.spec.neuron_archdir)
```
Actually, `spec` is not available in this function.